src/ui/utils/prefs.ts

```diff
@@ -66,6 +67,7 @@ export const features = new PrefsHelper("devtools.features", {
   unicornConsole: ["Bool", "unicornConsole"],
   showRedux: ["Bool", "showRedux"],
   enableLargeText: ["Bool", "enableLargeText"],
+  softFocus: ["Bool", "softFocus"],
```

Comment:
Ooo, should we go back and add this feature flag to #6915? I could open a PR that does that.